wiki/People/The_Team/Beatmap_Nominators/General_Information/en.md

```diff
@@ -1,18 +1,18 @@
 # General information for Beatmap Nominators
 
-Are you a [Beatmap Nominator](/wiki/People/Beatmap_Nominators), or interested in becoming one? If so, you have come to the right place! This article has all of the general information you will need to know as a new Beatmap Nominator.
+Are you a [Beatmap Nominator](/wiki/People/Beatmap_Nominators) *(BN)*, or interested in becoming one? If so, you have come to the right place! This article has all of the general information you will need to know as a new Beatmap Nominator.
 
 ## Important links
 
-- [Beatmap Nominator Rules](/wiki/People/Beatmap_Nominators/Rules): Outlines both how to undertake various actions, as well as how not to do things - you would want to avoid breaking these in any situation, so be sure to read them carefully! You can always ask a member of the [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team) if you are unsure about something, and they will be happy to help.
-- [Code of Conduct for Mapping and Modding](/wiki/Ranking_Criteria/Code_of_Conduct): As a Beatmap Nominator, you are expected to be an exemplary figure of the community, someone others look up to and respect, so misconduct may get you reprimanded or even expelled.
-- [Ranking Criteria](/wiki/Ranking_Criteria): If you are already a Beatmap Nominator, then you should be familiar with this. Essentially, it serves to explain what aspects of mapping are and are not allowed in the Ranked section, as well as including some general guidelines that people should try to follow in most cases. 
+- [Beatmap Nominator Rules](/wiki/People/Beatmap_Nominators/Rules): Outlines both how to undertake various actions, as well as how not to do things - you would want to avoid breaking these in any situation, so be sure to read them carefully! You can always ask a member of the [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team) *(NAT)* if you are unsure about something, and they will be happy to help.
```

Comment:
```suggestion
- [Beatmap Nominator Rules](/wiki/People/The_Team/Beatmap_Nominators/Rules): Outlines both how to undertake various actions, as well as how not to do things - you would want to avoid breaking these in any situation, so be sure to read them carefully! You can always ask a member of the [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) *(NAT)* if you are unsure about something, and they will be happy to help.
```